packages/components/Avatar/SPEC.md

```diff
@@ -83,8 +83,10 @@ The Avatar supports a circular and square (with rounded corners) shape.
 
 ### Sizes
 
-`Avatar` supports next sizes: `20, 24, 28, 32, 36, 40, 48, 56, 64, 72, 96, 120`.
-Default size for win32 is 24.
+`Avatar` supports the following sizes: `20, 24, 28, 32, 36, 40, 48, 56, 64, 72, 96, 120`.
+Android only supports the following sizes: `16, 20, 24, 32, 40, 56, 72`. Badge is not displayed for Size 16.
```

Comment:
nit: and AcitivtyRing

Reply:
ActivityRing is displayed for size16, only presenceBadge is disabled.